starclass/RFGCClassifier/RF_GC_featcalc.py

```diff
@@ -502,12 +525,15 @@ def freq_phasediffs(featdictrow, usedfreqs):
 		phase difference of 2nd to 1st and 3rd to 1st frequencies
 
 	"""
-	if len(usedfreqs) >= 2:
-		phi21 = featdictrow['phase'+str(usedfreqs[1]+1)] - 2*featdictrow['phase'+str(usedfreqs[0]+1)]
+	tab=featdictrow['frequencies']
+	if usedfreqs >= 2:
+		#phi21 = featdictrow['phase'+str(usedfreqs[1]+1)] - 2*featdictrow['phase'+str(usedfreqs[0]+1)]
+		phi21 = tab[(tab['num'] == 2) & (tab['harmonic']== 0)]['phase'] - 2*tab[(tab['num'] == 1) & (tab['harmonic']== 0)]['phase']
 	else:
 		phi21 = 0
-	if len(usedfreqs) >= 3:
-		phi31 = featdictrow['phase'+str(usedfreqs[2]+1)] - 3*featdictrow['phase'+str(usedfreqs[0]+1)]
+	if usedfreqs >= 3:
+		#phi31 = featdictrow['phase'+str(usedfreqs[2]+1)] - 3*featdictrow['phase'+str(usedfreqs[0]+1)]
+		phi31 = tab[(tab['num'] == 3) & (tab['harmonic']== 0)]['phase'] - 2*tab[(tab['num'] == 1) & (tab['harmonic']== 0)]['phase']
```

Comment:
You changed "3*" to "2*"... Don't know what is correct... But it has definitely changed...

Reply:
The original 3* was correct.